guides/common/modules/proc_deploying-the-ca-certificate-on-a-host-by-using-ansible-rex.adoc

```diff
@@ -0,0 +1,33 @@
+[id="deploying-the-ca-certificate-on-a-host-by-using-ansible-rex"]
+= Deploying the CA certificate on a host by using Ansible REX
+
+You can use remote execution (REX) with the Ansible provider to deploy the CA certificate.
+Redeploy the CA certificate when you change it on {ProjectServer}.
+
+.Prerequisites
+* The host is registered to {Project}.
+* Remote execution is enabled on the host.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Monitor* > *Jobs*.
+. Click *Run Job*.
+. From the *Job category* list, select `Ansible Commands`.
+. From the *Job template* list, select `Download and execute a script`.
+. Click *Next*.
+. Select hosts on which you want to execute the job.
+. In the *url* field, enter the following URL:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+http://_{foreman-example-com}_/unattended/public/foreman_ca_refresh
```

Comment:
```suggestion
https://_{foreman-example-com}_/unattended/public/foreman_ca_refresh
```

unless this is intentional because we assume that the cert is expired?

Reply:
We have two scenarios when we will need to run the script:
1. When the user prepares for the change in advance.
2. When the CA already expired/changed on the Foreman server.

I think the full procedure to replace the CA if the user prepares in advance (1) would be:
1. Add a new cert to Foreman's CA
2. Run this REX job to have the dual CA published to the host
3. Remove the old CA from Foreman
4. Run this REX job to replace the dual CA with only the new one.

In this case the `https` endpoint will still be valid.

In case the CA has already been replaced with a new cert (2), the host has no way to verify Foreman's identity, so the `https` endpoint will not work, and the only way to contact the server is with `http`.

Reply:
Perhaps I should add the general steps to docs as well...

Reply:
I've added a new procedure for CA cert renewal and used it as a recommendation.